includes/wikia/nirvana/WikiaSkin.class.php

```diff
@@ -398,6 +398,16 @@ public function getUserLoginStatusClass() {
 		return self::USER_LOGIN_STATUS_CLASS_ANON;
 	}
 
+	/**
+	 * VOLDEV-168: Add a community-specific class to the body tag
+	 * The class is the local DB name with a wiki- prefix
+	 *
+	 * @return string Generated CSS class for this wikia
+	 */
+	public function getBodyClassForCommunity() {
+		return "wiki-{$this->wg->DBname}";
```

Comment:
As a best practice, can you wrap this in `Sanitizer::escapeClass` to make sure it's a valid class just in case?